media-libs/netpbm/netpbm-10.86.27.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-inherit multilib toolchain-funcs
```

Comment:
Sorry if I asked this before: are we actually using multilib.eclass?

Reply:
I think we do, `pkgcheck scan` reports if we don't.  I just checked on another package:
```
UnusedInherits: version 1.3.5: unused eclass: multilib
```

Reply:
I removed it and got
```
IndirectInherits: version 10.86.27: multilib: indirect inherit usage: 'get_libname', line 58
```

Great tool :)